### PR TITLE
33942 - fix ps report error

### PR DESCRIPTION
**Pull request by jon-shipley**
Allow check.audit to be missing

### CODE REVIEW

File: functions-app/report-psychometrician/service/anomaly-report.service.js [hunk 1]
```diff
@@ -351,9 +355,15 @@ anomalyReportService.detectQuestionsThatWereShownForTooLong = (check) => {
   const audits = anomalyReportService.filterAllRealQuestionsAndPauseAudits(check)
   const config = check.data.config
   const head = R.head(audits)
+  if (!head) {
```
Comment (GeorgeChatzigiannis) [inline functions-app/report-psychometrician/service/anomaly-report.service.js]: From what I see you return when there's no single audit record.

Is that being handled properly afterwards when you return ?

Comment (jon-shipley) [reply functions-app/report-psychometrician/service/anomaly-report.service.js]: Yes.

We bail out of this test and move on to the next.

File: functions-app/report-psychometrician/service/anomaly-report.service.js [hunk 2]
```diff
@@ -351,9 +355,15 @@ anomalyReportService.detectQuestionsThatWereShownForTooLong = (check) => {
...
+    return
+  }
   const tail = R.tail(audits)
+  if (!tail) {
```
Comment (GeorgeChatzigiannis) [inline functions-app/report-psychometrician/service/anomaly-report.service.js]: @jon-shipley that can also mean that you return if you have a single item within the array of audits.
Please confirm if that is expected behavior

Comment (jon-shipley) [reply functions-app/report-psychometrician/service/anomaly-report.service.js]: Removed - thanks!

**Pull request merged**